test/boost/secondary_index_test.cc

```diff
@@ -711,11 +711,13 @@ SEASTAR_TEST_CASE(test_secondary_index_create_custom_index) {
         // "exceptions::invalid_request_exception: CUSTOM index requires
         // specifying the index class"
         assert_that_failed(e.execute_cql("create custom index on cf (a)"));
-        // It's also a syntax error to try to specify a "USING" without
-        // specifying CUSTOM. We expect the exception:
-        // "exceptions::invalid_request_exception: Cannot specify index class
-        // for a non-CUSTOM index"
-        assert_that_failed(e.execute_cql("create index on cf (a) using 'org.apache.cassandra.index.sasi.SASIIndex'"));
+        // This is the default syntax for specifying a custom index and we 
+        // support "opensearch_vector_index". This should work.
+        e.execute_cql("create custom index on cf (a) using 'opensearch_vector_index'").get();
```

Comment:
How do we know this command will always work? In the future, we may need to configure an external search engine node, and if we don't, this command will fail, no?

Personally, I think these opensearch tests should be separate test, in their separate file (and perhaps even test/cqlpy, not test/boost).